CHANGELOG.md

```diff
@@ -6,11 +6,13 @@
 
 * [CHANGE] Querier: returning 422 when query hits `max_fetched_chunks_per_query` and `max_fetched_series_per_query` limits in the store-gateway. #4056
 * [CHANGE] Packaging: Migrate FPM packaging solution to NFPM. Rationalize packages dependencies and add package for all binaries. #3911
+* [FEATURE] Ruler: added `keep_firing_for` support to alerting rules. #4099
```

Comment:
This also affects mimirtool. Should we mention it?

Reply:
Sure, done.